src/crepe/api/Camera.h

```diff
@@ -20,33 +21,35 @@ class Camera : public Component {
 	 * \param id Unique identifier for the camera component.
 	 * \param bg_color Background color for the camera view.
 	 */
-	Camera(game_object_id_t id, const Color & bg_color);
+	Camera(game_object_id_t id, const Color & bg_color, const ivec2 & screen,
+		   const ivec2 & viewport, const double & zoom, const vec2 & offset = {0, 0});
```

Comment:
waarom is dit  const vec2 & offset = {0, 0} en niet  const vec2 & offset?

Reply:
omdat je niet perse hoeft in te vullen